### ISSUE TITLE
this code can not be trained？

### ISSUE THREAD

**Issue by hnjzlishuang** [codemukul95/SVHN-classification-using-Tensorflow#1]
Hi, I tried to run this code, and the accuracy is still the same...Just like, all the time is 0.4, when I chose the learning rate is 0.1... Does this mean this network can not be optimized?

looking forward to your news. Thank you!

**Comment by jiaqizho** [comment]
this code is the office train code for the MNIST data. 
this project using this code try to slove the SVHN . 
so i run the code and test it even change some details , make it more reasonable 。 

but it play bad for SVHN .  you need to find anthor model to do it

**Comment by codemukul95** [comment]
@Smalinuxer  yeah did this project earlier......have implemented the modified network for SVHN will push the trainable and testing code soon.

**Comment by jiaqizho** [comment]
@hnjzlishuang 
@codemukul95 

i find someone writer an code . it work at SVHN, and it run out more than 90% rate .
the link : https://github.com/CreatCodeBuild/TensorFlow-Chinese-Tutorial/tree/master/16-19
He define two layer of CNN and two layer of full connect layer . and using the Regularization + Dropout + gradientDescent to update and it out very good.

you can see the code , it with english annotation , the writer also release the video which is chinese .

maybe help :)

**Comment by codemukul95** [comment]
HI @Smalinuxer  and @hnjzlishuang 
I have updated the code and ran it for 10 epochs
Achieved testing accuracy of 91.8%.
Kindly acknowledge if it helped.
